torch/csrc/dynamo/eval_frame.c

```diff
@@ -10,9 +10,14 @@
 #include <opcode.h>
 #include <stdbool.h>
 
+// The string has 26 static characters and 3 integers
+// Using int32 as max value for the integers they can be string of
+// len 10. Lastly, we need to add 1 for the null terminator:
+// 26 + 3 * 10 + 1 = 57
+#define MAX_TORCH_COMPILED_REGION_ANNOTATION_SIZE 57
```

Comment:
Why?!?! Just precompute it in the owning string that is going to be your const char* trace_id. Don't be copying strings in fastpath

Reply:
Oops not sure what I was thinking here. Removed!